app/decorators/movie_decorator.rb

```diff
@@ -1,79 +1,67 @@
 class MovieDecorator < Draper::Decorator
-  delegate_all
-
-  def release_date
-    handle_none object.release_date do
-      Time.parse(object.release_date).strftime("%B %Y")
-    end
+  def id
+    object["id"]
   end
 
-  def budget
-    handle_none object.budget do
-      h.number_to_currency(object.budget, precision: 0)
-    end
+  def title
+    object["title"]
   end
 
-  def runtime
-    handle_none object.runtime do
-      object.runtime
-    end
+  def overview
+    object["overview"]
   end
 
-  def vote_average
-    handle_none object.vote_average do
-      object.vote_average
-    end
+  def tagline
+    object["tagline"]
   end
 
-  def cast
-    CastDecorator.decorate_collection(credits.cast)
+  def poster_path
+    object["poster_path"]
   end
 
-  def cast_head
-    cast[0..4]
+  def genres
+    GenreDecorator.decorate_collection(object["genres"])
   end
 
-  def cast_tail
-    cast - cast_head
+  def youtube_trailers
+    TrailerDecorator.decorate_collection(object["trailers"]["youtube"])
   end
 
-  def crew
-    crew = CrewDecorator.decorate_collection(credits.crew)
-    crew = crew.group_by { |m| m.department }
-    test = crew.each do |department, members|
-      crew[department] = members.group_by { |m| m.job }
+  def belongs_to_collection
+    if object["belongs_to_collection"].present?
+      CollectionDecorator.decorate(object["belongs_to_collection"])
     end
-
-    test
   end
 
-  def crew_head
-    crew.select { |department, _| department == 'Directing' }
+  def release_date
+    if object["release_date"].present?
+      Time.parse(object["release_date"]).strftime("%B %Y")
+    end
   end
 
-  def crew_tail
-    crew.select { |department, _| department != 'Directing' }
+  def budget
+    unless object["budget"].to_i.zero?
+      h.number_to_currency(object["budget"], precision: 0)
+    end
   end
 
-  def poster_url(size: 'w342')
-    if poster_path
-      "#{h.configuration.base_url}#{size}#{poster_path}"
-    else
-      "http://dummyimage.com/342x513/d9d9d9/000000.png&text=N/A"
+  def runtime
+    unless object["runtime"].to_i.zero?
+      object["runtime"]
     end
   end
 
-  def trailers
-    object.trailers.youtube[0...2]
+  def vote_average
+    unless object["vote_average"].to_f.zero?
+      object["vote_average"]
+    end
   end
 
-  private
-
-  def handle_none(value)
-    if value.present? && value != 0
-      yield
+  def poster_url(size: 'w342')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.